ops/systemd/export_workers.env

```diff
@@ -0,0 +1,32 @@
+## General settings
+HOSTNAME=https://export.hotosm.org
+DJANGO_SETTINGS_MODULE=core.settings.project
+
+## SMTP Settings
+EMAIL_HOST_PASSWORD=<SECURE_SMTP_PASSWORD>
+EMAIL_HOST=smtp.mailgun.org
+EMAIL_HOST_USER=export@hotosmmail.org
+
+## Storage options
+EXPORT_DOWNLOAD_ROOT=/mnt/data/downloads
+EXPORT_STAGING_ROOT=/mnt/data/staging
+PLANET_FILE=/mnt/data/planet/planet.osm.pbf
+
+## Deprecated options; Soon to be removed
+# GARMIN_MKGMAP=/usr/local/mkgmap/mkgmap.jar
+# GARMIN_SPLITTER=/usr/local/splitter/splitter.jar
+# GENERATOR_TOOL=/usr/local/bin/generator_tool
+# OSMCONVERT=/usr/bin/osmconvert
+# OSMAND_MAP_CREATOR_DIR=/usr/local/OsmAndMapCreator
+
+## HDX Options
+SYNC_TO_HDX=True
+HDX_API_KEY=<SECURE_HDX_API_KEY>
+HDX_NOTIFICATION_EMAIL=osm2hdx@googlegroups.com
+HDX_SITE=prod
+
+## Overpass options
+OVERPASS_API_URL=https://overpass-export.hotosm.org
```

Comment:
We need to add rawdataapi url also 
RAW_DATA_API_URL=

Reply:
made the change